qml/Main.qml

```diff
@@ -32,6 +33,20 @@ PageStackWindow {
     }
 
     property string windowTitle: util.currentWindowTitle();
+    property bool physicalKeyboardOpen : false
+
+    ContextProperty {
+        key: "maemo/InternalKeyboard/Open"
+        onValueChanged: {
+            if (value == 1) {
+                physicalKeyboardOpen = true
+                window.setTextRenderAttributes()
+            } else {
+                physicalKeyboardOpen = false
+                window.setTextRenderAttributes();
```

Comment:
Nitpick: property declaration has extra space before colon, this has inconsistent semicolon with previous branch.

Also a bit more declarative way would be defining property as bound to value of this. Plus then "onPhysicalKeyboardOpenChanged: window.setText...()"